app/models/idea.rb

```diff
@@ -19,7 +19,7 @@ def submitted?
   end
 
   def approved_by_admin?
-    return false if status == 'awaiting_approval' || status.nil?
+    return false if status == 'awaiting_approval' || status == 'draft'
```

Comment:
`status == 'awaiting_approval' || status == 'draft'`
Could be rewritten as
`awaiting_approval? || draft?`